src/formatter.ts

```diff
@@ -52,6 +52,13 @@ export interface AlignmentContexts<T> {
   resolutionMultiplier: number;
 }
 
+export interface AlignmentModifierContexts {
```

Comment:
Seems like we can merge this with the above, no?

Reply:
See my comment above.